packages/graphql/src/types/index.ts

```diff
@@ -330,7 +331,7 @@ export type NodeSubscriptionsEvent =
               old: undefined;
               new: Record<string, any>;
           };
-          id: number;
+          id: number; // | string
```

Comment:
what do these comments mean?

Reply:
These types are not correct, as these types may be strings. I'll have to update these for CDC, but I haven't fixed it yet. I can remove these comments for now